docs/_ext/djangodocs.py

```diff
@@ -109,8 +109,9 @@ def warner(msg):
                                                    warn=warner,
                                                    linenos=linenos,
                                                    **highlight_args)
+    css_classes = ['highlight-%s' % lang, 'clippify']
     starttag = self.starttag(node, 'div', suffix='',
-                             CLASS='highlight-%s' % lang)
+                             CLASS=' '.join(css_classes))
```

Comment:
Was a previous iteration of the code more complicated? Not sure the intermeidate `css_classes` variable has much benefit.